en/editor/project/index.md

```diff
@@ -169,16 +169,20 @@ Example:
 
 ## Physics
 
-![Physics](./index/physics-index.png)
-
 This physics configuration will be take effect only when the **Physics Module** is enabled in the **Engine Module**.
 
+It should be noted that the collision matrix is a further encapsulation of the physical group mask functionality.It is similar to the grouping configuration of 'Creator 2D', but with some differences. The grouping configuration of 'Creator 2D' is only responsible for initialization, while the grouping configuration of 'Creator 3D' is also responsible for automatic updates (which may be adjusted to be initialization only in the future).
+
+Therefore, after `useCollisionMatrix` is turned on, the mask value can only be set through the collision matrix, meaning that related apis such as `setMask` cannot be used. To modify the collision matrix at runtime, refer to [PhysicsSystem](../../physics/physics-system.md#Interfaces) of `setCollisionGroup`.
```

Comment:
To modify the collision matrix at runtime, refer to [PhysicsSystem](../../physics/physics-system.md#Interfaces) of `setCollisionGroup`.
-->
To modify the collision matrix at runtime, please refer to the [PhysicsSystem](../../physics/physics-system.md#Interfaces) documentation under `setCollisionGroup`.